spec/facades/user_campaigns_with_characters_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require_relative '../../app/facades/user_campaigns_with_characters'
+
+describe 'UserCampaignsWithCharacters' do
+
+  it 'Should have respond to user' do
+    character = FactoryGirl.create :character
+    user = character.user
+    campaigns_characters = UserCampaignsWithCharacters.new(user)
+    expect(campaigns_characters.user).to eql user
+  end
+
+  it 'Should respond to characters' do
+    user = FactoryGirl.create :user
+    characters = FactoryGirl.create_list :character, 3, user: user
+    campaigns_characters = UserCampaignsWithCharacters.new(user)
+    expect(campaigns_characters.characters).to match_array characters
```

Comment:
You're not actually testing the class responds to `characters` method, you're testing characters returned by the class are the same as the ones assigned to the user in the first place. Change the spec description